src/Export/Adapters/DescriptionAdapter.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Export\Adapters;
+
+use FINDOLOGIC\Export\Data\Description;
+use FINDOLOGIC\FinSearch\Utils\Utils;
+use Shopware\Core\Content\Product\ProductEntity;
+
+class DescriptionAdapter
+{
+    public function adapt(ProductEntity $product): ?Description
+    {
+        $value = new Description();
+        $description = $this->getDescription($product);
+
+        if (!$description) {
+            return null;
+        }
+
+        $value->setValue($description);
+
+        return $value;
```

Comment:
I think we should stay consistent here, you are creating the `Description` class just before setting the value in the `DateAddedAdapter`.
```suggestion
        if (!$descriptionValue = $this->getDescription($product)) {
            return null;
        }

        $description = new Description();
        $description->setValue($descriptionValue);

        return $description;
```